script/events.js

```diff
@@ -503,7 +503,41 @@ var Events = {
 		$('<div>').addClass('clear').appendTo(lootButtons);
 		if(lootButtons.children().length > 1) {
 			lootButtons.appendTo(desc);
+			var takeAll = new Button.Button({
+				id: 'loot_take_all',
+				text: _('Take All'),
+				click: Events.takeAllLoot
+			}).addClass('take-all-button').appendTo(lootButtons);
+		}
+	},
+
+	takeAllLoot: function(){
+
+		var stoppedEarly = false;
+		$('#lootButtons')
```

Comment:
If I'm reading this right, this button simulates a single click on each loot button. That means it'll still require multiple clicks to truly "take all," no?